### PR TITLE
more careful getModuleAt

### PR DESCRIPTION
**Pull request by pfitzseb**
Fixes https://github.com/julia-vscode/LanguageServer.jl/issues/767.

I'm not sure when this can happen, but should fix the crash regardless.

### CODE REVIEW

File: src/requests/features.jl [hunk 1]
```diff
@@ -304,7 +304,13 @@ function julia_getModuleAt_request(params::TextDocumentPositionParams, server::L
     doc = getdocument(server, URI2(params.textDocument.uri))
     offset = get_offset(doc, params.position)
     x = get_expr(getcst(doc), offset)
-    return x isa EXPR ? get_module_of(StaticLint.retrieve_scope(x)) : "Main"
+    if x isa EXPR
+        scope = StaticLint.retrieve_scope(x)
+        if x !== nothing
```
Comment (ZacLN) [inline src/requests/features.jl:309]: Is this supposed to be `scope !== nothing` ?

Comment (pfitzseb) [reply src/requests/features.jl]: Whoops, yes, of course.

**Comment by davidanthoff** [comment]
@ZacLN should I merge this?

**Comment by davidanthoff** [comment]
Did @ZacLN review this? I think with the StaticLint stuff it would be good if he looked over things before we merge.

**Comment by ZacLN** [comment]
HI, yes I had already made comments - all good

**Pull request merged**